containers/docker/dev/ubuntu/base/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ubuntu:bionic
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+		ca-certificates       \
+		wget                  \
+                libaprutil1           \
```

Comment:
is there a centralized list of deps for HPCC on bionic? If there is, let's use it to build up this list. If there isn't lets consider it so that one list can be used for this purpose and for documentation

Reply:
no

Reply:
The only place we mentioned is HPCC Platform wiki "Build HPCC".

Reply:
And for different branch the list will be updated, for example 7.2.x to 7.4.x

Reply:
There is a repo to provision our build servers: https://github.com/xwang2713/openstack-ansible/tree/master/provisions.
But I usually take it as a template and modify before run the provision

Reply:
@Michael-Gardner is there any reason why we don't list all necessary dep packages in cmak_modules/dependencies (other than the fact most of them are default in the target distro)? Are there any disadvantages to explicitly listing all requirements?

Reply:
Since the package names probably different in different OS cmake module usually only search for the file name or pattern. Also HPCC-Platform provide find module cmake files under cmake_modules they are all header file/lib file (depends on OS) based search not package names